src/loader/loader.c

```diff
@@ -672,9 +694,11 @@ _PG_init(void)
 	prev_post_parse_analyze_hook = post_parse_analyze_hook;
 	/* register shmem startup hook for the background worker stuff */
 	prev_shmem_startup_hook = shmem_startup_hook;
+	prev_shmem_request_hook = shmem_request_hook;
```

Comment:
Shouldn't these hooks only be registered with PG >= 15 (other versions seems to use the function call in Line 654)?

Reply:
fixed